<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WIFIMode</class>
 <widget class="QDialog" name="WIFIMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(27, 150, 208);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>801</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QTextBrowser" name="textBrowser">
            <property name="minimumSize">
             <size>
              <width>360</width>
              <height>360</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <property name="horizontalSpacing">
             <number>6</number>
            </property>
            <property name="verticalSpacing">
             <number>12</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 20pt &quot;Sans Serif&quot;;</string>
              </property>
              <property name="text">
               <string>设备编号</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="ClickedQLabel" name="MachID_lb">
              <property name="minimumSize">
               <size>
                <width>211</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>211</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border:1px solid green;
font: 18pt &quot;宋体&quot;;

border-color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string>1009</string>
              </property>
              <property name="buddy">
               <cstring>MachID_lb</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>WIFI名称</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="lWIFIName_le">
              <property name="minimumSize">
               <size>
                <width>211</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>211</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border:1px solid green;
border-color: rgb(255, 255, 255);

font: 18pt &quot;宋体&quot;;</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>WiFi密码</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="ClickedQLabel" name="WIFIPassword_le">
              <property name="minimumSize">
               <size>
                <width>211</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>211</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border:1px solid green;
border-color: rgb(255, 255, 255);

font: 18pt &quot;宋体&quot;;</string>
              </property>
              <property name="text">
               <string>15968867217</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>IP地址</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="ClickedQLabel" name="IPAdress_le">
              <property name="minimumSize">
               <size>
                <width>211</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>211</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border:1px solid green;
border-color: rgb(255, 255, 255);

font: 18pt &quot;宋体&quot;;</string>
              </property>
              <property name="text">
               <string>112.16.93.229</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_4">
              <property name="styleSheet">
               <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>端口号</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="ClickedQLabel" name="port_le">
              <property name="minimumSize">
               <size>
                <width>211</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>211</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border:1px solid green;
border-color: rgb(255, 255, 255);

font: 18pt &quot;宋体&quot;;</string>
              </property>
              <property name="text">
               <string>8282</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>41</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 24pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>自动连接(F0)</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="remoteControl_cb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>41</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 24pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>远程设置(F1)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="autoConn_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>41</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>连接WiFi(F2)</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="back_pb">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>101</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>返    回</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickedQLabel</class>
   <extends>QLabel</extends>
   <header>clickedqlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
